projects/plugins/jetpack/extensions/blocks/ai-assistant/ai-control.js

```diff
@@ -0,0 +1,164 @@
+import {
+	Button,
+	Icon,
+	MenuGroup,
+	MenuItem,
+	NavigableMenu,
+	Popover,
+	TextareaControl,
+} from '@wordpress/components';
+import { useRef } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import {
+	arrowRight,
+	check,
+	image,
+	pencil,
+	postContent,
+    postExcerpt,
+	title,
+	undo,
+} from '@wordpress/icons';
+import Loading from './loading';
+
+const AIControl = ( {
+	aiType,
+	animationDone,
+	content,
+	contentIsLoaded,
+	getSuggestionFromOpenAI,
+	handleAcceptContent,
+	handleGetSuggestion,
+	isSelected,
+	isWaitingState,
+	loadingImages,
+	placeholder,
+	retry,
+	setAiType,
+	setUserPrompt,
+	showRetry,
+} ) => {
+	const handleInputEnter = event => {
+		if ( event.key === 'Enter' && ! event.shiftKey ) {
+			event.preventDefault();
+			handleGetSuggestion();
+		}
+	};
+
+	const toggleAIType = () => {
+		if ( aiType === 'text' ) {
+			setAiType( 'image' );
+		} else {
+			setAiType( 'text' );
+		}
+	};
+
+	const popoverContainer = useRef( null );
+	return (
+		<>
+			<div className="jetpack-ai-assistant__input-wrapper">
+				{ ( ( ! content && isWaitingState ) || loadingImages ) && <Loading /> }
+				<TextareaControl
+					onChange={ value => setUserPrompt( value ) }
+					onKeyPress={ handleInputEnter }
+					rows="1"
+					placeholder={ isWaitingState ? __( 'AI writing', 'jetpack' ) : placeholder }
+					className="jetpack-ai-assistant__input"
+				/>
+				<div className="jetpack-ai-assistant__controls">
+					<Button
+						onClick={ () => handleGetSuggestion() }
+						isSmall={ true }
+						disabled={ isWaitingState }
+						label={ __( 'Do some magic!', 'jetpack' ) }
+					>
+						<Icon icon={ arrowRight } />
+					</Button>
+				</div>
+			</div>
+			<div ref={ popoverContainer } style={ { marginTop: '8px' } }>
```

Comment:
Let's try to place these in the block toolbar

Reply:
Moved controls in [35eaea3](https://github.com/Automattic/jetpack/pull/30498/commits/35eaea3ef10c710df454b4e4f9b66af4022b7fa3)